### DISCUSSION TITLE
Change URL from localhost:9090 to something nicer eg: opengrok:9090

### DISCUSSION THREAD

**Discussion by karthikrathnam** [oracle/opengrok#4205] [Q&A] [answered]
Is it possible to change the URL instead of using localhost or hostname to something else eg: opengrok:9090?

**Comment by vladak** [comment, marked as answer]
As long as you have the naming infrastructure setup (e.g. DNS) to resolve the `opengrok` hostname to the particular IP address (and preferably back), there should be no problem with that.
